src/E0_evaluation.py

```diff
@@ -39,7 +51,7 @@ def evaluate_dict(dict_values, results_main, results_meta):
             "kpi": {
                 "cost_matrix": pd.DataFrame(
                     columns=[
-                        "label",
+                        LABEL,
                         "costs_total",
                         "costs_om",
                         "costs_investment",
```

Comment:
This is already post-processing, but we could also make those keys constant variables:
"kpi", "cost_matrix", "cost_total", "cost_om", "cost_investment", "scalar_matrix", "optimizedAddCap", "total_flow", "annual_total_flow"

Reply:
7693b1c3db0a3bdbff53594580f42abc4071a090